build/Build.cs

```diff
@@ -0,0 +1,1045 @@
+                        ReplaceTextInFiles(tentacleInstallerWixProject, "{WixTasksPath}", wixNugetPackagePath / "tools" / "wixtasks.dll");
+                        
+                        MSBuildTasks.MSBuild(settings => settings
+                            .SetConfiguration("Release")
+                            .SetProperty("AllowUpgrade", "True")
+                            .SetVerbosity(MSBuildVerbosity.Normal) //TODO: Set verbosity from command line argument
+                            .SetTargets("build")
+                            .SetTargetPlatform(platform)
+                            .SetTargetPath(RootDirectory / "installer" / "Octopus.Tentacle.Installer" / "Octopus.Tentacle.Installer.wixproj"));
+                    }
+                    catch (Exception e)
+                    {
+                        Console.WriteLine(e);
+                        throw;
+                    }
+                    finally
+                    {
+                        restoreWiXProjectAction.Invoke();
+                    }
+                    
+                    var builtMsi = RootDirectory / "installer" / "Octopus.Tentacle.Installer" / "bin" / platform / "Octopus.Tentacle.msi";
+                    SignAndTimeStamp(builtMsi);
+                    
+                    var platformString = platform == MSBuildTargetPlatform.x64 ? "-x64" : "";
+                    MoveFile(
+                        builtMsi,
+                        ArtifactsDirectory / "msi" / $"Octopus.Tentacle.{OctoVersionInfo.FullSemVer}{platformString}.msi");
+                });
+            }
+
+            // This is a slow operation
+            var wixNugetInstalledPackage = NuGetPackageResolver.GetLocalInstalledPackage("wix", ToolPathResolver.NuGetPackagesConfigFile);
+            if (wixNugetInstalledPackage == null) throw new Exception("Failed to find wix nuget package path");
+
+            EnsureExistingDirectory(ArtifactsDirectory / "msi");
+            PackWindowsInstallers(MSBuildTargetPlatform.x64, wixNugetInstalledPackage.Directory);
+            PackWindowsInstallers(MSBuildTargetPlatform.x86, wixNugetInstalledPackage.Directory);
+        });
+
+    // ReSharper disable InconsistentNaming
+    Target PackCrossPlatformBundle => _ => _
+        .Description("Packs the cross-platform Tentacle.nupkg used by Octopus Server to dynamically upgrade Tentacles.")
+        // Disabling these dependencies right now until we understand how we can consume artifacts from previous build steps without having to re-run this target
+        // .DependsOn(PackWindows)
+        // .DependsOn(PackLinux)
+        // .DependsOn(PackOSX)
+        .Executes(() =>
+        {
+            string ConstructDebianPackageFilename(string packageName, string architecture) {
+                var filename = $"{packageName}_{OctoVersionInfo.FullSemVer}_{architecture}.deb";
+                return filename;
+            }
```

Comment:
Would this read better as 
```suggestion
            string ConstructDebianPackageFilename(string packageName, string architecture) => $"{packageName}_{OctoVersionInfo.FullSemVer}_{architecture}.deb";
```